src/main/java/redis/clients/jedis/CommandObjects.java

```diff
@@ -800,6 +800,14 @@ protected CommandArguments commandArguments(ProtocolCommand command) {
         .key(dstKey).add(from).add(to).add(timeout), BuilderFactory.STRING);
   }
 
+  public final CommandObject<Map<String, List<String>>> lmpop(ListDirection from, String... keys) {
+    return new CommandObject<>(commandArguments(LMPOP).add(keys.length).keys((Object[]) keys).add(from), BuilderFactory.SEARCH_SYNONYM_GROUPS);
+  }
+
+  public final CommandObject<Map<String, List<String>>> lmpop(ListDirection from, int count, String... keys) {
+    return new CommandObject<>(commandArguments(LMPOP).add(keys.length).keys((Object[]) keys).add(from).add(COUNT).add(count), BuilderFactory.SEARCH_SYNONYM_GROUPS);
+  }
```

Comment:
IMO, we can create a new `KeyedListElements` similar to `KeyedListElement` by using using `List<String> elements` instead of `String element` and use that as return type here. [1]

Reply:
added